sdk/mediaservices/azure-resourcemanager-mediaservices/src/main/java/com/azure/resourcemanager/mediaservices/models/MediaServicesOperationResultsGetHeaders.java

```diff
@@ -28,6 +29,18 @@ public final class MediaServicesOperationResultsGetHeaders {
     @JsonProperty(value = "Location")
     private String location;
 
+    // HttpHeaders containing the raw property values.
+    /**
+     * Creates an instance of MediaServicesOperationResultsGetHeaders class.
+     *
+     * @param rawHeaders The raw HttpHeaders that will be used to create the property values.
+     */
+    public MediaServicesOperationResultsGetHeaders(HttpHeaders rawHeaders) {
```

Comment:
@weidongxu-microsoft Should we allow this and add breaking change log for it, since it's introduced by codegen?

Reply:
We can add a changelog, but skip it in pom.xml, as it should be output-only (and it is from an LRO polling operation that no one would call) so likely no user will call the ctor.

Reply:
We once considered whether we should not generate that LRO poll operation group at all, then found that it was already generated for mediaservices.

Reply:
Got it. I think we can skip changelog then.